utils.py

```diff
@@ -0,0 +1,137 @@
+import ast
+import os
+import random
+import hashlib
+import hmac
+import pytz
+
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+from pytz import timezone
+
+slack_signing_secret = os.environ["SLACK_SIGNING_SECRET"]
+team_list_var = os.environ["TEAM_LIST"]
+
+
+def create_markdown_list(elements):
+    if not elements:
+        return ""
+
+    # Slack doesn't support actual markdown lists so I'm mocking a list
+    # with a "-" char before each element and a new line after
+    # https://api.slack.com/reference/surfaces/formatting#block-formatting
+    markdown_list = ""
```

Comment:
can you build this list with the `join` function?

```
separator = '\n'
return separator.join(elements)
```

Reply:
I think to get something Slack would interpret as a `<ul>` we would need `'\n-'.join(elements)`, but we would also need to prefix the _first_ element with a `-` in that case 😅. Could definitely do it either way, and I think it's mostly just a stylistic thing, although using `join` is probably a teensy bit more "Pythonic" if we care about that, haha.